controller/config.go

```diff
@@ -391,18 +399,28 @@ func Validate(c Config) error {
 		}
 	}
 
-	if v, ok := c[JujuHASpace].(string); ok {
-		if !names.IsValidSpace(v) {
-			return errors.NewNotValid(nil, "invalid juju HA space name")
-		}
+	if err := validateSpaceConfig(c, JujuHASpace, "juju HA"); err != nil {
+		return err
```

Comment:
if we are doing it elsewhere in the file we should consider using:
return errors.Trace(err)

That gives us stack traces.